lhotse/audio.py

```diff
@@ -277,6 +280,7 @@ def __init__(self, base_audio: np.ndarray, sampling_rate: int):
         self.tracks = [base_audio]
```

Comment:
RE "point of reference in terms of offset", is there a way to make that more precise?  It's not clear to me why users would want to specify the energy via a file, and whether it's a good idea for all the output to have the same average energy.  (Systems sometimes are not robust w.r.t. energy variations).

Reply:
oh, perhaps the AudioMIxer is to be instantiated for each piece of audio to be mixed.  Might be good to make that clear?

Reply:
You are right in the latter comment. I've updated the description of both Audio and Feature mixers to clarify.